packages/core/admin/utils/get-plugins-path.js

```diff
@@ -1,15 +1,29 @@
 'use strict';
 
-const { join, resolve } = require('path');
+const { join, resolve, sep, posix } = require('path');
 const fs = require('fs-extra');
 // eslint-disable-next-line import/no-extraneous-dependencies
 const glob = require('glob');
 
 // Only for dev environement
 const getPluginsPath = () => {
   const rootPath = resolve(__dirname, '..', join('..', '..', '..', 'packages'));
-  const corePath = join(rootPath, 'core', '*');
-  const pluginsPath = join(rootPath, 'plugins', '*');
+  /**
+   * So `glob` only supports '/' as a path separator, so we need to replace
+   * the path separator for the current OS with '/'. e.g. on windows it's `\`.
+   *
+   * see https://github.com/isaacs/node-glob/#windows for more information
+   *
+   * and see https://github.com/isaacs/node-glob/issues/467#issuecomment-1114240501 for the recommended fix.
+   */
+  let corePath = join(rootPath, 'core', '*');
+  let pluginsPath = join(rootPath, 'plugins', '*');
+
+  if (process.platform === 'win32') {
+    corePath = corePath.split(sep).join(posix.sep);
+    pluginsPath = pluginsPath.split(sep).join(posix.sep);
+  }
```

Comment:
Could you add a test for this branch (by mocking `process.platform`), because I imagine this will easily be forgotten about ...?

Reply:
I can try, have you checked the tests for this? They're a bit odd. They rely on actually using path and we only run tests in a macOS env, so I'm not sure the best way? Any ideas?

Reply:
Suppose I could mock path for the test?

Reply:
I haven't looked, but can try to suggest something if you want next week?